script/aria.js

```diff
@@ -172,12 +172,14 @@ require(["core/pubsubhub"], function( respecEvents ) {
                     propList[title] = { is: type, title: title, name: content, desc: desc, roles: [] };
                     var abstract = container.querySelector("." + type + "-applicability");
                     if ((abstract.textContent || abstract.innerText) === "All elements of the base markup") {
-                        globalSP.push({ is: type, title: title, name: content, desc: desc, prohibited: false });
-                    } 
+                        globalSP.push({ is: type, title: title, name: content, desc: desc, prohibited: false });	                        globalSP.push({ is: type, title: title, name: content, desc: desc, prohibited: false, deprecated: false });
```

Comment:
Is this missing a newline? Or was it intended to replace the original line (and just add the `, deprecated: false`?

Reply:
good catch.... thanks